articles/overview/understanding-quantum-computing.md

```diff
@@ -0,0 +1,65 @@
+
+The quantum material behaves according to the laws of quantum mechanics, leveraging concepts such as probabilistic computation, superposition, and entanglement. These concepts provide the basis for quantum algorithms that harness the power of quantum computing to solve complex problems. This article describes some of the essential concepts of quantum mechanics on which quantum computing is based.
+
+## A bird’s-eye view of quantum mechanics
+
+Quantum mechanics, also called quantum theory, is a branch of physics that deals with particles at the atomic and subatomic levels. At the quantum level, however, many of the laws of mechanics you take for granted don’t apply. Superposition, quantum measurement, and entanglement are three phenomena that are central to quantum computing.  
+
+### Superposition vs. binary computing
+
+Imagine that you are exercising in your living room. You turn all the way to your left and then all the way to your right. Now turn to your left and your right at the same time. You can’t do it (not without splitting yourself in two, at least).  Obviously, you can’t be in both of those states at once – you can’t be facing left and facing right at the same time.
+
+However, if you are a quantum particle, then you can be *facing left* AND *facing right* AND *anywhere in-between* at the same time due to a phenomenon known as **superposition** (also known as **coherence**).
+
+A quantum particle such as an electron has its own “facing left or facing right” property called **spin**, referred to as either up or down, or to make it more relatable to classical binary computing, let’s just say 1 or 0. When a quantum particle is in a superposition state, it’s a linear combination of an infinite number of states between 1 and 0, but you don’t know which one it will be until you actually look at it, which brings up our next phenomenon, **quantum measurement**.
+
+### Quantum measurement
+
+Now, let’s say your friend comes over and wants to take a picture of you exercising. Most likely, they’ll get a blurry image of you turning somewhere between all the way left and all the way right.
+
+But if you’re a quantum particle, an interesting thing happens. No matter where you are when they take the picture, it will always show you either all the way left or all the way right – nothing in-between.
+
+This is because the act of observing or measuring a quantum particle **collapses** the superposition state (also known as **decoherence**) and the particle takes on a classical binary state of either 1 or 0.
+
+This binary state is helpful to us, because in computing you can do lots of things with 1’s and 0’s. However, once a quantum particle has been measured and collapsed, it stays in that state forever (just like your picture) and will always be a 1 or 0. As you’ll see later, though, in quantum computing there are operations that can “reset” a particle back to a superposition state so it can be used for quantum calculations again.
```

Comment:
I want to avoid misleading readers like every other quantum computing intro. Superposition is not |0> and |1> at the same time. It is a linear combination of |0> and |1>. Interference is a more accepted interpretation than "collapse". Say the quantum particle is in a state sqrt(0.4)|0>+sqrt(0.6)|1>, i.e. you have 40% time facing left and 60% time facing right. Your friend takes 100 pictures. 40 times you are facing left and 60 times you are facing right.